src/test/scala/autoversion/AutoVersionSpec.scala

```diff
@@ -0,0 +1,61 @@
+package autoversion
+
+import org.scalatest.{FreeSpec, Matchers}
+import sbtrelease.Version.Bump
+import sbtrelease.{Version, versionFormatError}
+
+class AutoVersionSpec extends FreeSpec with Matchers {
```

Comment:
Would it be possible to switch to FlatSpec instead, to avoid mixing testing styles ?

Reply:
Sure thing!